orc8r/cloud/docker/docker-compose.yml

```diff
@@ -15,6 +15,7 @@ services:
 #      DATABASE_SOURCE: "magma_dev:magma_dev@(maria)/magma_dev"
       SQL_DIALECT: psql
       SERVICE_REGISTRY_MODE: yaml
+      VERSION_TAG: DEV
```

Comment:
Here and elsewhere: sometimes "dev" can be synonymous with "master branch". Maybe consider something like "local-dev"